pkg/api/v1/reporter/reporter_test.go

```diff
@@ -19,6 +21,15 @@ var _ = Describe("Reporter", func() {
 		reporter                               rep.Reporter
 		mockResourceClient, fakeResourceClient clients.ResourceClient
 	)
+
+	BeforeEach(func() {
+		Expect(os.Setenv("POD_NAMESPACE", "gloo-system")).NotTo(HaveOccurred())
```

Comment:
Why do we set this? Is this normally set by k8s?

Reply:
Yes, it comes from k8s, we plumb it through in our deployment templates, i.e.: https://github.com/solo-io/gloo/blob/master/install/helm/gloo/templates/1-gloo-deployment.yaml#L169-L172